setup.py

```diff
@@ -34,8 +34,8 @@
       include_package_data=True,
       zip_safe=True,
       install_requires=[
-          "requests>=2.9.1",
-          "lxml>=3.2.3"
+          "requests>=2.26.0",
+          "lxml>=4.6.3"
```

Comment:
Is this bump up needed specifically for Python 3

Reply:
Hari, in Python3 environment which is prerequisite for blackjack as mentioned in the work-item, its expecting upgraded versions of these packages. So i upgraded them to recent versions.